nuget.config

```diff
@@ -4,6 +4,6 @@
     <clear />
```

Comment:
it looks there are binary .nupkg files in this PR.  These should be the build output from the projects, so I don't think they should be checked into the Git repository, unless I'm missing something.

Reply:
Nuget packages which were added in bin folder is a temporary solution to fix CI build issue. Once the Package get published it will be referred from Nuget.org

Reply:
@NiranjanVirtuosity why are the projects using a binary PackageReference instead of a source ProjectReference?  Typically code in the same solution would use a direct source reference.

Reply:
There's a version conflict between the newly added ManagedBatchParser and these projects when we used project reference, So we decided to refer from NuGet.